make-util.js

```diff
@@ -536,6 +536,24 @@ var getExternals = function (externals, destRoot) {
             copyGroups(package.cp, packageSource, destRoot);
         });
     }
+
+    // for any file type that has to be shipped with task
+    if (externals.hasOwnProperty('files')) {
+        var files = externals.files;
+        files.forEach(function (file) {
+            assert(file.url, 'file.url');
+            assert(file.dest, 'file.dest');
+
+            // download the file from url
+            var fileSource = downloadFile(file.url);
+            var fileName = file.url.substr(file.url.lastIndexOf("/") + 1);
```

Comment:
i would rather the dest define the file name instead.

interpreting file name from url has a limitation. for example, consider an fwlink